content-src/asrouter/templates/Trailhead/_Trailhead.scss

```diff
@@ -1,9 +1,19 @@
+
+
 .trailhead {
-  $benefit-icon-size: 64px;
-  $benefit-icon-spacing: $benefit-icon-size + 20px;
+  $benefit-icon-size: 62px;
+  $benefit-icon-spacing: $benefit-icon-size + 12px;
 
   background: url('#{$image-path}trailhead/accounts-form-bg.jpg') 100% 100%;
+  background-size: cover;
```

Comment:
The `100% 100%` is actually the positioning (not the size), and it should be `bottom`. Then you can put the `cover` size on the same line: `background: url(…) bottom / cover;`